style.css

```diff
@@ -173,10 +175,10 @@ h1 > i {
 }
 
 .input_field {
- -ms-flex: 1;
-	 -webkit-box-flex: 1;
-	 -webkit-flex: 1;
-		 flex: 1;
+ 	-webkit-box-flex: 1;
+	-webkit-flex: 1;
+	    -ms-flex: 1;
```

Comment:
You don't change the order of CSS properties just because it looks good somewhere. :P 
Everything else is fine in this PR but @MrSaqibAmeen if you can put `-ms-flex: 1;` back at the top, I can approve it. Since it is the least important and I want it to be checked in the end.